feature/client/src/main/java/com/mifos/feature/client/clientDetails/di/RepositoryModule.kt

```diff
@@ -0,0 +1,20 @@
+package com.mifos.feature.client.clientDetails.di
+
+import com.mifos.core.network.datamanager.DataManagerClient
+import com.mifos.feature.client.clientDetails.data.repository_imp.ClientDetailsRepositoryImp
+import com.mifos.feature.client.clientDetails.domain.repository.ClientDetailsRepository
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+
+@Module
+@InstallIn(SingletonComponent::class)
+object RepositoryModule {
```

Comment:
This should be in data module. feature module doesn't need to know from where data is coming. They only care about consuming the data.

Reply:
Yes, I have created one repository and repository-imp folder inside data module to hold this repository